src/components/Sorting.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { Button, Form } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import * as api from "../api.js";
+
+function Sorting() {
+  const [, setSortBy] = useState([]);
+  const [isAsc, setIsAsc] = useState(false);
+
+  useEffect(() => {
+    api.fetchCategories().then((sortBy) => {
+      setSortBy(sortBy);
+    });
+  }, [setSortBy]);
+
+  function handleOrderChange() {
+    setIsAsc(!isAsc);
+  }
+  return (
+    <div>
+      <div>
+        <p>Order by:</p>
+        <Link to={`?order=${isAsc ? "asc" : "desc"}&sort_by=votes`}>
+          <Button>Sort By Votes count</Button>
+        </Link>
+        <Link to={`?order=${isAsc ? "asc" : "desc"}&sort_by=created_at`}>
+          <Button>Sort By date</Button>
```

Comment:
dropdowns would be much more user friendly than buttons here, especially for the sorting